dev-manual/api/_code/hide_request_ingest.curl

```diff
@@ -0,0 +1,4 @@
+curl -X DELETE \
+    http://my_archivematica_instance.archivematica.org/api/ingest/ \
+    89fd478d-c896-461a-baec-5d1faf661020/delete/ \
+    -H 'Authorization: ApiKey [_your_username_]:[_your_ApiKey_]'
```

Comment:
Let's see how the `trailing-whitespace` `pre-commit` hook I'm introducing in https://github.com/artefactual/archivematica-docs/pull/455 modifies these `*.curl` files.

Reply:
:facepalm: I meant the [end-of-file-fixer](https://github.com/pre-commit/pre-commit-hooks/blob/main/README.md#end-of-file-fixer) which I introduced later. I just rebased your branch on top of the latest `1.15` and it fixed this correctly.

Reply:
Thanks